camelus_lib.ml

```diff
@@ -524,52 +567,52 @@ module FormatUpgrade = struct
        OpamStd.String.Map.is_empty replace_files
     then Lwt.return None
     else
-    let%lwt _ =
-      RepoGit.git gitstore
-        ["reset"; "-q"; "--mixed"; if merge then onto else head]
-    in
-    let%lwt () =
-      Lwt_list.iter_s (fun (path, contents) ->
-          match contents with
-          | Some contents ->
-            let%lwt hash =
-              RepoGit.git gitstore ["hash-object"; "-w"; "--stdin"] ~input:contents
-              >|= String.trim
-            in
-            let%lwt _ =
-              RepoGit.git gitstore
-                ["update-index"; "--ignore-missing"; "--add";
-                 "--cacheinfo"; "100644,"^hash^","^path]
-            in
-            Lwt.return_unit
-          | None ->
-            let%lwt _ =
-              RepoGit.git gitstore
-                ["update-index"; "--ignore-missing"; "--remove"; "--"; path]
-            in
-            Lwt.return_unit)
-        (OpamStd.String.Map.bindings replace_files)
-    in
-    let%lwt tree = RepoGit.git gitstore ["write-tree"] >|= String.trim in
-    let committer = git_identity () in
-    let env = [|
-      "GIT_AUTHOR_NAME="^ author.Git.User.name;
-      "GIT_AUTHOR_EMAIL="^ author.Git.User.email;
-      "GIT_COMMITTER_NAME="^ committer.Git.User.name;
-      "GIT_COMMITTER_EMAIL="^ committer.Git.User.email;
-    |] in
-    let message =
-      message (OpamPackage.Set.elements
-                 (OpamPackage.Set.union packages removed_packages))
-    in
-    RepoGit.git gitstore ~env
-      ("commit-tree" ::
-       "-m" :: message ::
-       (if merge then ["-p"; onto] else []) @
-       [ "-p"; head;
-         tree ])
-    >|= String.trim
-    >|= fun hash -> Some (hash, message)
+      let%lwt _ =
```

Comment:
Please upgrade your `ocp-indent` :)

Reply:
:-)

Reply:
(or check that you have `strict_else=auto` configured)

Reply:
My bad, I wasn't using default ocp-indent config and overlooked adding the proper `.ocp-indent` file. Fixing my mess and pushing to the 2.0 branch :)